docs/_ext/towncrier_draft_ext.py

```diff
@@ -106,6 +106,21 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     return ' '.join(msg_chunks)
 
 
+def _nodes_from_rst(state: statemachine.State, rst: str) -> List[nodes.Node]:
+    """Turn an RST string into a node that can be used in the document."""
+    node = nodes.Element()
+    node.document = state.document
+    nested_parse_with_titles(
+        state=state,
+        content=statemachine.ViewList(
+            rst.splitlines(),
```

Comment:
Is this better than `statemachine.string2lines()`?

Reply:
no idea.

Reply:
http://code.nabla.net/doc/docutils/api/docutils/statemachine/docutils.statemachine.string2lines.html